test/jest/cookieSyncManager.spec.ts

```diff
@@ -496,60 +406,73 @@ describe('CookieSyncManager', () => {
                 (mockImage as unknown) as HTMLImageElement
             );
 
-            const loggerSpy = jest.fn();
-
             const mockMPInstance = ({
                 _Store: {
                     webviewBridgeEnabled: false,
                     pixelConfigurations: [pixelSettings],
                 },
                 _Persistence: {
+                    // FIXME: why is this here?  it doesn't exist in the SDK (it is used elsewhere too)
```

Comment:
Is this still relevant?

Reply:
as we discussed, i have removed this comment as well as the line in question below it and elsewhere